cmd/bacalhau/docker_run.go

```diff
@@ -142,6 +150,10 @@ func init() { //nolint:gochecknoinits,funlen // Using init in cobra command is i
 		&ODR.OutputVolumes, "output-volumes", "o", ODR.OutputVolumes,
 		`name:path of the output data volumes. 'outputs:/outputs' is always added.`,
 	)
+	dockerRunCmd.PersistentFlags().StringVar(
```

Comment:
Please remove. This is not required. You should just print the jobspec to stdout.